src/main/java/se/sundsvall/workflow/businesslogic/worker/MyWorker.java

```diff
@@ -14,28 +11,28 @@
 import org.zalando.problem.Problem;
 import org.zalando.problem.Status;
 
-import se.sundsvall.workflow.integration.camunda.handler.FailureHandler;
+import se.sundsvall.workflow.businesslogic.handler.FailureHandler;
 
 @Component
-@ExternalTaskSubscription("printMessage")
-public class PrintMessageWorker implements ExternalTaskHandler {
-	private static final Logger LOGGER = LoggerFactory.getLogger(PrintMessageWorker.class);
+@ExternalTaskSubscription("myWorker")
+public class MyWorker implements ExternalTaskHandler {
```

Comment:
Extend AbstractWorker?

Reply:
There is no need as the AbstractWorker only implements the method clearUpdateAvailable (which is not used in this worker).